### PR TITLE
Validations

### PR DESCRIPTION
**Pull request by kaitcham**
For this branch, I followed these [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md)

### CODE REVIEW

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @kaitcham 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
 
 - Nothing is being displayed when you visit `localhost:3000` because you didn't implement the routes in the `config/route.rb` file. Please work on it to test validation.
  
image
 
 - Good job, all the tests are passing successfully.
 image
 
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kaitcham** [comment]
Hi @Emmyn5600 

Thank you for reviewing my project. But can you please clarify the changes you are requesting? I don't think there are in the project requirements

File: README.md [hunk 1]
```diff
@@ -1,24 +1,131 @@
-# README
```
Comment (arslanbisharat) [inline README.md:1]: - [ ] As requested by the previous code reviewer, please add a proper config file.

**Review by nicupop729 — COMMENTED** [review]
Hello @kaitcham 👋,

_This is an additional review as a result of the second opinion request._

Very nice implementations of the project requirements!
Congratulations on that 👏 

Regarding the requests from the students, you are rights, what they ask from you is not listed in the project requirements.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @kaitcham, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**